lib/api.ts

```diff
@@ -100,9 +100,14 @@ export function getContributorBySlug(file: string, detail = false) {
     ...activityData,
     activity: [...activityData.activity, ...getSlackMessages(data.slack)],
   };
+  let AllUnqPrReviews = new Set();
 
-  const weightedActivity = activityData.activity.reduce(
+  let weightedActivity = activityData.activity.reduce(
     (acc, activity) => {
+      if (activity.type == "pr_reviewed") {
+        AllUnqPrReviews.add(activity.title);
```

Comment:
is there no `id` instead of title?

Reply:
![image](https://github.com/coronasafe/leaderboard/assets/101407963/e4703a6a-fc95-40f3-b9e6-984595243593)
no

Reply:
![image](https://github.com/coronasafe/leaderboard/assets/101407963/3e0689c8-1194-4083-ad97-7c324b23bd8a)

yes